packages/cli/README.md

```diff
@@ -737,9 +749,10 @@ const minigit = Command.make(
     Option.match(configs, {
       onNone: () => Console.log("Running 'minigit'"),
       onSome: (configs) => {
-        const keyValuePairs = Array.from(configs)
-          .map(([key, value]) => `${key}=${value}`)
-          .join(", ")
+        const keyValuePairs = Array.from(
+          configs,
```

Comment:
`.map` is not needed here. It just allocates a useless array.

[MDN `Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#mapfn)